spec.bs

```diff
@@ -260,8 +260,22 @@ dictionary AuctionAdInterestGroupKey {
 The <dfn for=Navigator method>leaveAdInterestGroup(group)</dfn> method steps
 are:
 
+1. If [=this=]'s [=relevant global object=]'s [=associated Document=] is not [=allowed to use=] the
+  "[=join-ad-interest-group=]" [=policy-controlled feature=], then [=exception/throw=] a
+  "{{NotAllowedError}}" {{DOMException}}.
+1. Let |ownerUrl| be the result of running the [=URL parser=] on
+    |group|["{{AuctionAdInterestGroupKey/owner}}"].
+1. If |ownerUrl| is an error [=exception/throw=] a {{TypeError}}.
+1. Set |owner| to |ownerUrl|'s [=url/origin=].
+1. Set |name| to |group|["{{AuctionAdInterestGroupKey/name}}"].
 1. Let |p| be [=a new promise=].
-1. TODO... the rest (referencing |p| here so the build tool isn't sad).
+1. Let |queue| be the result of [=starting a new parallel queue=].

```

Comment:
The queue maybe is maybe a bit heavy here if you never have to reference it again. You can just do the following if you want

```
1. Run these steps in parallel:
    1. TODO:
    1. Queue a task [...]
    1. Remove [...]
1. Return |p|.
```

Reply:
Done, thanks.